src/System.Globalization/tests/Common/Unicode_5_1/PropList.txt

```diff
@@ -0,0 +1,1207 @@
+# PropList-5.1.0.txt
+# Date: 2008-03-20, 17:55:27 GMT [MD]
+#
+# Unicode Character Database
+# Copyright (c) 1991-2008 Unicode, Inc.
+# For terms of use, see http://www.unicode.org/terms_of_use.html
```

Comment:
And this readme.txt and PropList.txt isn't required for the tests. Should I just remove them?